CollectNewspaperKarel.java

```diff
@@ -13,16 +13,22 @@
 public class CollectNewspaperKarel extends SuperKarel {
 
 	// You fill in this part
-	// Added a comment
-	public void run(){
-		
+	private void goToDoor(){
 		move();
 		turnRight();
 		move();
 		turnLeft();
 		move();
 		move();
+	}
+	
+	private void collectNewsPeper(){
+		
 		pickBeeper();
+		
+	}
+	
+	private void turnBackHome(){
 		turnRight();
 		turnRight();
 		move();
```

Comment:
Can we write a decomposition for moves that need to be done consecutively?

Think of moveNTimes? Will that make code cooler?